src/List/web/List.js

```diff
@@ -1,10 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import ListItem from './ListItem';
 
-const List = styled.ul`
- ${''}
+const styles = {
+  listStyleType(props) {
+    if (props.type === 'ul') {
+      return 'disc';
+    } else if (props.type === 'ol') {
+      return 'decimal';
+    }
+    return 'none';
+  },
+};
+
+const List = styled(({
+  type,
+  ...props
+}) => React.createElement(props.type || 'ul', props))`
+  margin: 0;
+  padding: ${(props) => props.type ? props.theme.px([0, 0, 0, 2]) : '0'};
+  list-style-type: ${styles.listStyleType};
+  list-style-position: ${(props) => props.type ? 'outside' : ''};
+
+  > ${/* sc-selector */ListItem} {
+    padding: ${(props) => props.type ? props.theme.px([1, 1, 1, 0]) : props.theme.px(2)};
+  }
 `;
 
+List.propTypes = {
+  type: PropTypes.oneOf(['ul', 'ol']),
```

Comment:
Last we discussed, type was one of `unordered` and `ordered`, any reason it's `ul` and `ol` instead?